desktop/cmp/input/Checkbox.js

```diff
@@ -47,7 +49,7 @@ Checkbox.propTypes = {
     displayUnsetState: PT.bool,
 
     /** Alignment of the inline label relative to the control itself, default right. */
```

Comment:
might also change this docstring to be more straightforward -- the use of the word alignment seems a bit confusing.